Set default rootless sigstore (#1035)
containers/image: docs/containers-registries.d.5.md

`docker.io/library/busybox`, the configuration for `docker.io` is ignored
(even if some element of the configuration is defined for `docker.io` and not for `docker.io/library/busybox`).

> ### Signature Storage

Reviewer: Maybe
```suggestion
### Built-in Defaults
```
everything is about signature storage in this file.

> If default-docker not configured and no matched configuration applied for the container image can be found,

Reviewer: (Maybe something like
```suggestion
If no `docker` section can be found for the container image, and no `default-docker` section is configured
```
to go from the most-specific to the most general, in the “order of evaluation”)

> the default directory, `/var/lib/containers/sigstore` or `$HOME/.local/share/containers/sigstore`,  will be used for reading and writing signatures.

Reviewer: This should say how the value is chosen.

   This key is optional; if it is missing, no signature storage is defined (no signatures
   are download along with images, adding new signatures is possible only if `sigstore-staging` is defined).

The default `sigstore-staging` is `/var/lib/containers/sigstore` from `/etc/containers/registries.d/default.yaml`.

> If the user has no privilege to write the signature to the default sigstore-staging, the YAML format files

Reviewer: That’s not the actual condition.

> under `$HOME/.config/containers/registries.d` can be configured, otherwise, `$HOME/.local/share/containers/sigstore` will be used.

Reviewer: All of this should be in the ”Registries, Scopes and Search Order” section, maybe in an extra named subsection.

> ## Examples

Reviewer: Consider adding an extra “example” for no configuration.
